src/main/scala/Core/instructions.scala

```diff
@@ -258,9 +258,18 @@ object CSRs {
   val clear_ipi = 0x50f
   val core_id = 0x510
   val reset = 0x51d
-  val tohost = 0x51e
+  //val tohost = 0x51e
   val fromhost = 0x51f
```

Comment:
?

Reply:
If the comment was just regarding the fact that the tohost register is commented out, the line has been removed now :)